ui/app/src/components/ItemDisplay/ItemDisplay.tsx

```diff
@@ -40,7 +40,7 @@ export interface ItemDisplayProps<LabelTypographyComponent extends React.Element
 	showNavigableAsLinks?: boolean;
 	classes?: Partial<Record<ItemDisplayClassKey, string>>;
 	sxs?: PartialSxRecord<ItemDisplayClassKey>;
-	item: ContentItem;
+	item: LightItem & Partial<ContentItem>;
```

Comment:
This doesn't make a great deal of sense. ContentItem inherits from LightItem. What are you trying to do here? Perhaps we should declare what's known to be required internally.

Reply:
Updated to be of type `LightItem | ContentItem`, with updated assertions in the component. That way both types of items are supported, and the component handles the rendering depending on the props it has